modules/ui/src/main/scala/lucuma/ui/aladin/ReactAladin.scala

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.aladin
+
+import cats.syntax.all.*
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.hooks.*
+import japgolly.scalajs.react.hooks.Hooks.UseState
+import japgolly.scalajs.react.vdom.TopNode
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.math.*
+import lucuma.react.common.*
+import lucuma.ui.aladin.facade.*
+import org.scalajs.dom.html
+
+import scala.scalajs.js
+
+type Aladin = lucuma.ui.aladin.facade.JsAladin
+
+extension (a: Aladin)
+  def size: Size = Size(a.getSize()(0), a.getSize()(1))
+
+  def fov: Fov =
+    Fov(Angle.fromDoubleDegrees(a.getFov()(0)), Angle.fromDoubleDegrees(a.getFov()(1)))
+
+  def onPositionChangedCB(cb: PositionChanged => Callback): Callback =
+    Callback(
+      a.on("positionChanged", (o: JsPositionChanged) => cb(PositionChanged.fromJs(o)).runNow())
+    )
+
+  def onZoomCB(cb: Fov => Callback): Callback =
+    Callback(a.on("zoomChanged", (_: Double) => cb(fov).runNow()))
+
+  def onZoomCB(cb: => Callback): Callback =
+    Callback(a.on("zoomChanged", (_: Double) => cb.runNow()))
+
+  def onFullScreenToggleCB(cb: Boolean => Callback): Callback =
+    Callback(a.on("fullScreenToggled", (t: Boolean) => cb(t).runNow()))
+
+  def onFullScreenToggleCB(cb: => Callback): Callback =
+    Callback(a.on("fullScreenToggled", (_: Boolean) => cb.runNow()))
+
+  def onMouseMoveCB(cb: MouseMoved => Callback): Callback =
+    Callback(a.on("mouseMove", (t: JsMouseMoved) => cb(MouseMoved.fromJs(t)).runNow()))
+
+  def pixelScale: PixelScale =
+    PixelScale(a.getSize()(0) / a.getFov()(0), a.getSize()(1) / a.getFov()(1))
+
+  def increaseZoomCB: Callback =
+    Callback(a.increaseZoom())
+
+  def decreaseZoomCB: Callback =
+    Callback(a.decreaseZoom())
+
+  def fixLayoutDimensionsCB: Callback =
+    Callback(a.fixLayoutDimensions())
+
+  def recalculateViewCB: Callback =
+    Callback(a.recalculateView())
+
+  def requestRedrawCB: Callback =
+    Callback(a.requestRedraw())
+
+  def gotoRaDecCB(c: Coordinates): Callback =
+    Callback(a.gotoRaDec(c.ra.toAngle.toDoubleDegrees, c.dec.toAngle.toSignedDoubleDegrees))
+
+case class ReactAladin(
```

Comment:
This part is new, maybe worth a more detailed review

Reply:
Seems fine to me.